src/test/unit/sync_http/test_sync_hook.py

```diff
@@ -532,6 +532,21 @@ def lambda_test_function(event, context):
     ]
 
 
+def test_not_jsonable_return(monkeypatch):
+    @lumigo_tracer()
+    def lambda_test_function(event, context):
+        return {"a": open("/tmp/a.txt", "wb")}
```

Comment:
If the test will run on the machine without `"/tmp"` the test will fail.
You can use another object that isn't json serializable and isn't dependant on the machine running the date, e.g `datetime.datetime`.